openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -55,7 +55,7 @@
 the entire cookie and use it to impersonate the victim.
 
 """
-from __future__ import absolute_import
+from __future__ import absolute_import, unicode_literals
```

Comment:
Again, are we sure the blanket conversion is safe here?

Reply:
Reviewed this one as well, should be okay I believe.